config-linux.md

```diff
@@ -148,5 +134,60 @@ rootfsPropagation sets the rootfs's mount propagation. Its value is either slave
 
 ## Security
 
-**TODO:** security profiles
+### Linux capabilities
+
+Capabilities is an array that specifies Linux capabilities that can be provided to the process
+inside the container. Valid values are the string after `CAP_` for capabilities defined 
+in [the man page](http://man7.org/linux/man-pages/man7/capabilities.7.html)
+
+```json
+   "security": {
+       "capabilities": [
+            "AUDIT_WRITE",
+            "KILL",
+            "NET_BIND_SERVICE"
+        ]
+    }
+```
+
+### Selinux process label
+
+Selinux process label specifies the label with which the processes in a container are run.
+For more information about SELinux, see  [Selinux documentation](http://selinuxproject.org/page/Main_Page)
```

Comment:
A more specific link would be nice.  It might be obvious to SELinux users (I'm not one), but the main page didn't say anything about process labels.